apps/climatemappedafrica/src/components/HowItWorks/index.js

```diff
@@ -0,0 +1,200 @@
+              [theme.breakpoints.up("md")]: {
+                position: "relative",
+                top: 0,
+                height: theme.typography.pxToRem(524),
+                padding: `${theme.typography.pxToRem(66)} ${theme.typography.pxToRem(
+                  77,
+                )} ${theme.typography.pxToRem(69)} 0`,
+              },
+              [theme.breakpoints.up("lg")]: {
+                backgroundColor: theme.palette.background.default,
+                height: theme.typography.pxToRem(600),
+                opacity: 0.9,
+                padding: `${theme.typography.pxToRem(81)} ${theme.typography.pxToRem(98)}`,
+              },
+            }}
+          >
+            <Box
+              sx={{
+                position: "relative",
+                height: theme.typography.pxToRem(227),
+                width: "100%",
+                "& .video-js": {
+                  width: "100%",
+                  height: "100%",
+                },
+                "& .vjs-poster": {
+                  backgroundColor: "#ffffffE6",
+                  backgroundSize: "120%",
+                },
+                "& .video-js .vjs-big-play-button": {
+                  display: "none",
+                },
+                [theme.breakpoints.up("md")]: {
+                  height: theme.typography.pxToRem(194),
+                  width: theme.typography.pxToRem(299),
+                  "& .vjs-poster": {
+                    backgroundSize: "auto",
+                  },
+                },
+                [theme.breakpoints.up("lg")]: {
+                  height: theme.typography.pxToRem(244),
+                  width: theme.typography.pxToRem(376),
+                },
+              }}
+            >
+              <Player {...video} />
+            </Box>
+            <Typography
+              sx={{
+                marginTop: theme.typography.pxToRem(18),
+              }}
+              variant="h4"
+            >
+              {title}
+            </Typography>
+            <RichText
+              elements={description}
+              sx={{
+                fontFamily: theme.typography.body1.fontFamily,
+                margin: `${theme.typography.pxToRem(16.5)} 0`,
+                color: theme.palette.grey.dark,
+                [theme.breakpoints.up("lg")]: {
+                  width: theme.typography.pxToRem(278),
+                },
+              }}
+            />
+            <Button href={link.href} variant="text">
+              {link.label}
+            </Button>
+          </Grid>
+          <Grid item lg={1} />
+          <Grid
+            item
+            xs={12}
+            md={5}
+            sx={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
+              marginBottom: theme.typography.pxToRem(23),
+              [theme.breakpoints.up("md")]: {
+                justifyContent: "flex-end",
+                marginBottom: 0,
```

Comment:
Avoid using margins with `Grid` items. `Grid` is implemented using [negative margins](https://mui.com/material-ui/react-grid/#negative-margin) and setting on margins will conflict with that.